data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/v1/beacon/GetBlock.java

```diff
@@ -62,7 +62,9 @@ public GetBlock(final ChainDataProvider chainDataProvider, final JsonProvider js
       method = HttpMethod.GET,
       summary = "Get block",
       tags = {TAG_BEACON},
-      description = "Retrieves block details for given block id.",
+      description =
+          "Retrieves block details for given block id.\n\n"
+              + "__NOTE__: only phase 0 blocks are returned, use `/eth/v2/beacon/blocks/{block_id}` for altair blocks.",
```

Comment:
This implies only altair blocks come v2.  Struggling to find the right wording. Maybe
```suggestion
              + "__NOTE__: only phase 0 blocks are returned, use `/eth/v2/beacon/blocks/{block_id}` for multiple milestone support.",
```